docs/configuration_file.md

```diff
@@ -65,6 +68,28 @@ Where:
 
 DaLI comes with a few builtin plugins, but more are needed: help us make DaLI a robust open-source, community-driven crypto data loader by [contributing](../CONTRIBUTING.md#contributing-to-the-repository) plugins for exchanges and wallets!
 
+### Binance.com Section (REST)
+This plugin is REST-based and requires setting up API Keys in your Binance.com account settings (click on the API link).
```

Comment:
This line is the same as the Coinbase one: just wanted to make sure this is not a copy/paste error and that Binance.com also has an "API" link in the settings.

Reply:
I clarified this a bit, thanks.